### ISSUE TITLE
Fix the package.json dependencies

### ISSUE THREAD

**Issue by chasenlehara** [bitovi-components/bit-c3#27]
- [ ] `can` should also be in `peerDependencies`
- [ ] `cssify` should be moved to `devDependencies`

**Comment by justinbmeyer** [comment]
Can should probably be a straight `dependency`.  `peerDependencies` are deprecated.  Also, you can make SEMVER point at every supported CanJS version.

Before moving `cssify`, I'd make sure browserify apps work without it.  If I remember correctly, it's needed.

**Comment by leoj3n** [comment]
[Here is a good, quick read](https://codingwithspike.wordpress.com/2016/01/21/dealing-with-the-deprecation-of-peerdependencies-in-npm-3/) on the deprecation of `peerDependencies`.

Related to the necessity of browserify-specific configuration in `package.json`:

https://github.com/donejs/generator-donejs/issues/134

Feels bad to make all CanJS packages depend on `cssify`, just to be compatible with browserify. I wonder if it's really necessary; but never used browserify so not really sure where to begin to test.

Is there an issue wherefrom this idea of including `cssify` as a dependency originated from?

**Comment by justinbmeyer** [comment]
@leoj3n it's only necessary for plugins that load CSS.

**Comment by leoj3n** [comment]
@chasenlehara I think this issue can be closed now?

**Comment by chasenlehara** [comment]
I don’t think this can be closed. I moved `can` to `devDependencies` and I’m unsure if it should be moved to `peerDependencies`. `cssify` is still listed in `dependencies` so that hasn’t been fixed.
